src/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -644,10 +645,7 @@ public override bool Equals(object obj)
         }
 
         public override int GetHashCode()
-        {
-            return unchecked(value.GetHashCode() ^
-                    state.GetHashCode() ^
-                    knownColor.GetHashCode());
-        }
+            => HashHelpers.Combine(
+                    HashHelpers.Combine(value.GetHashCode(), state.GetHashCode()), knownColor.GetHashCode());
```

Comment:
I realised that in the unknown name case we can just use the name alone as the hash code, so am doing so now.